sysml.library/Domain Libraries/Quantities and Units/UnitsAndScales.sysml

```diff
@@ -304,7 +426,7 @@ package UnitsAndScales {
 	}
 
 	/**
-	 * Declare a constraint to verify that the given unit power factors comply with the required quantity dimension
+	 * Constraint definition to verify that the given unit power factors comply with the required quantity dimension
 	 *
 	 * TODO: The implementation of this constraint requires reflection.

```

Comment:
Probly should remove TODOs.

Reply:
I suppose we can leave this one "TODO" in here for now, since it marks an incomplete definition that should be completable before we are done. But if it does not get done before submission, then the whole definition will need to be removed.